SonarQube.TeamBuild.PreProcessor/Roslyn/ActiveRule.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SonarQube.TeamBuild.PreProcessor.Roslyn.Model
+{
+    public class ActiveRule
+    {
+        public string RepoKey { set; get; }
+        public string RuleKey { set; get; }
+        public string TemplateKey { set; get; }
```

Comment:
Not sure we need that since there are no rule templates in C# and VB.NET Roslyn analyzers. Do we have rule template in FXCop @tvsonar ?